dotnet-projects/StyleCop/StyleCop/Properties/launchSettings.json

```diff
@@ -0,0 +1,41 @@
+﻿{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "iisSettings": {
+    "windowsAuthentication": false,
+    "anonymousAuthentication": true,
+    "iisExpress": {
+      "applicationUrl": "http://localhost:54516",
+      "sslPort": 44348
+    }
+  },
+  "profiles": {
+    "http": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "applicationUrl": "http://localhost:5021",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
```

Comment:
From now on we should only keep https profile, so this can be deleted.

Reply:
@amuslic,
the http profile hasn't been deleted yet. 
We should only keep https, so both http (this one not yet) and IIS Express profile (this one already done) should be deleted